packages/components/src/avatar/src/AnonymousAvatar.tsx

```diff
@@ -0,0 +1,70 @@
+import { Box } from "../../box";
+import { ComponentProps, forwardRef } from "react";
+import {
+    InternalProps,
+    OmitInternalProps,
+    SlotProps,
+    StyledComponentProps,
+    cssModule,
+    mergeProps,
+    normalizeSize,
+    slot
+} from "../../shared";
+import { ResponsiveProp, useResponsiveValue } from "../../styling";
+import { AnonymousIcon } from "./svgs";
+
+const DefaultElement = "div";
+
+export interface InnerAnonymousAvatarProps extends SlotProps, InternalProps, StyledComponentProps<typeof DefaultElement | "image"> {
+    /**
+     * The label to display for the avatar.
```

Comment:
i think other places where we have the aria-label like this, the documentation is : 

```
    /**
     * See [WCAG](https://www.w3.org/TR/WCAG20-TECHS/ARIA14.html).
     */
```